ci-operator/config/openshift/procfs/OWNERS

```diff
@@ -0,0 +1,15 @@
+component: "Monitoring"
+
+reviewers:
```

Comment:
we should add @dgrisonnet

Reply:
unfortunately he can't be added for now.